src/Sarif.Converters/AndroidStudioConverter.cs

```diff
@@ -70,30 +69,14 @@ public override void Convert(Stream input, IResultLogWriter output, OptionallyEm
                 Name = "AndroidStudio"
             };
 
-            var fileInfoFactory = new FileInfoFactory(null, dataToInsert);
-            Dictionary<string, FileData> fileDictionary = fileInfoFactory.Create(results);
-
             var run = new Run()
             {
-                Tool = tool,
-                ColumnKind = ColumnKind.Utf16CodeUnits
+                Tool = new Tool { Name = ToolFormat.AndroidStudio },
```

Comment:
>Tool [](start = 27, length = 4)

Did you also mean to remove the local variable `tool`?

Reply:
This is marked "Resolved" but the local variable `tool` on Line 67 is still there.

---
In reply to: [244519901](https://github.com/Microsoft/sarif-sdk/pull/1191#discussion_r244519901) [](ancestors = 244519901)